packages/cli/templates/default-backend-plugin/src/plugin.ts.hbs

```diff
@@ -0,0 +1,25 @@
+/import { loggerToWinstonLogger } from '@backstage/backend-common';
```

Comment:
Extra prefix slash?

Reply:
Just a typo from my other window 😅 Thanks for the catch